src/util/rest/data/count.js

```diff
@@ -1,2 +1,3 @@
-module.exports = (call, idx) => call("POST", idx, { endpoint: "_count" })
-  .then(r => (r.statusCode === 200 ? r.body.count : false));
+module.exports = (call, idx) => call("POST", `${idx}@*`, { endpoint: "_count" })
+  // eslint-disable-next-line no-underscore-dangle
+  .then(r => (r.statusCode === 200 && r.body._shards.total !== 0 ? r.body.count : false));
```

Comment:
Does it make sense to use a `get(r, 'body._shards.total')`? You'd escape the `no-underscore-dangle` and avoid a 502 if `_shards` is not returned